manifests/components/cert-manager.jsonnet

```diff
@@ -44,11 +44,104 @@ local CERT_MANAGER_IMAGE = (import "images.json")["cert-manager"];
+          "name": "Issuer",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "JSONPath": ".status.conditions[?(@.type==\"Ready\")].message",
+          "name": "Status",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "description": "CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC.",
+          "JSONPath": ".metadata.creationTimestamp",
+          "name": "Age",
+          "type": "date",
+        },
+      ],
+      names+: {
+        shortNames+: ["cert", "certs"],
+      },
+    },
+  },
+
+  issuerCRD: kube.CustomResourceDefinition("certmanager.k8s.io", "v1alpha1", "Issuer") {
+    spec+: {
+      additionalPrinterColumns+: [
+        {
+          "JSONPath": ".status.state",
+          "name": "State",
+          "type": "string",
+        },
+        {
+          "JSONPath": ".spec.issuerRef.name",
+          "name": "Issuer",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "JSONPath": ".status.reason",
+          "name": "Reason",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "description": "CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC.",
+          "JSONPath": ".metadata.creationTimestamp",
+          "name": "Age",
+          "type": "date",
+        },
+      ],
+    },
+  },
 
+  orderCRD: kube.CustomResourceDefinition("certmanager.k8s.io", "v1alpha1", "Order") {
+    spec+: {
+      additionalPrinterColumns+: [
+        {
+          "JSONPath": ".status.state",
+          "name": "State",
+          "type": "string",
+        },
+        {
+          "JSONPath": ".spec.issuerRef.name",
+          "name": "Issuer",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "JSONPath": ".status.reason",
+          "name": "Reason",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "description": "CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC.",
+          "JSONPath": ".metadata.creationTimestamp",
+          "name": "Age",
+          "type": "date",
+        },
+      ],
+    }
   },
 
-  issuerCRD: kube.CustomResourceDefinition("certmanager.k8s.io", "v1alpha1", "Issuer"),
+  challengeCRD: kube.CustomResourceDefinition("certmanager.k8s.io", "v1alpha1", "Challenge"),
```

Comment:
By reading https://github.com/jetstack/cert-manager/blob/release-0.6/deploy/manifests/cert-manager.yaml (which I believe this jsonnet fragment is based on), I don't see the `additionalPrinterColumns` spec I see on the YAML.

Reply:
fixed